index.js

```diff
@@ -67,6 +67,9 @@ module.exports = {
   // FastBoot. This one is returned here as default configuration in order to make it
   // available at run time.
   config: function (environment, runConfig) {
+    if (process.env.DISABLE_CSP) {
```

Comment:
I had to return after setting `_runConfig`, because https://github.com/rwjblue/ember-cli-content-security-policy/blob/master/lib/utils.js#L96 was null (`Cannot read properties of null (reading 'contentSecurityPolicy')`)
```js
this._runConfig = runConfig;

if (process.env.DISABLE_CSP) {
  return {};
}
```